.travis.yml

```diff
@@ -34,12 +34,11 @@ env:
 # call OpenRA to check for YAML errors
 # Run the NUnit tests
 script:
- - travis_retry make all-dependencies
  - make all
  - test "$TRAVIS_OS_NAME" == "linux" && make check || echo "Skipping check"
  - test "$TRAVIS_OS_NAME" == "linux" && make check-scripts || echo "Skipping scripts check"
  - test "$TRAVIS_OS_NAME" == "linux" && make test || echo "Skipping tests"
- - test "$TRAVIS_OS_NAME" == "linux" && make nunit || echo "Skipping nunit tests"
+ - test "$TRAVIS_OS_NAME" == "linux" && ~/.nuget/packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe --noresult OpenRA.Test.nunit || echo "Skipping nunit tests"
```

Comment:
Changing this to the following seems to fix it, but I don't understand why...
```yaml
 - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
     mono ~/.nuget/packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe --noresult OpenRA.Test.dll;
   fi
```
`mono ~/.nuget/packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe --noresult OpenRA.Test.nunit` works fine on my machine, and the `.nunit` file used to be required for it to know where to import `OpenRA.Mods.Common.dll` 😕

Reply:
#10325 and #10326 provide context on the .nunit file. It may no longer be relevant with nunit3 and msbuild toolchain.